src/sentry/grouping/enhancer/actions.py

```diff
@@ -142,7 +141,6 @@ class VarAction(Action):
     _VALUE_PARSERS: dict[str, Callable[[Any], Any]] = {
         "max-frames": int,
         "min-frames": int,
-        "invert-stacktrace": get_rule_bool,
```

Comment:
Are we sure there are no customer rules using this?

Reply:
I removed the rules from the enhancement-config file.
If any customer uses it we can remove it from the stack trace rules once it gets deployed (I just did it from projects using `sentinel` or `prefix` parser actions).